global-news-app/src/components/Search/Search/Search.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { SearchContainer } from "./Search.styles";
+import SearchInput from "../SearchInput/SearchInput";
+import Select from "components/Select/Select";
+
+import RecentSearchesDropdown from "../RecentSearchesDropdown/RecentSearchesDropdown";
+import { ClickAwayListener } from "@mui/base/ClickAwayListener";
+import { SearchSelectOptions } from "Utils/MockData";
+
+const Search = () => {
+  const [option, setOption] = React.useState("");
+  const [searchHistory, setSearchHistory] = React.useState<string[]>([]);
+  const [showDropdown, setShowDropdown] = React.useState(false);
+
+  const handleSearch = (query: string) => {
```

Comment:
- remove the comments, the code seems pretty self-explanatory
- consider adding `TODO: ` for something that you are planning to finish later
- you can install todo highlight vs code extension

![Screenshot 2024-03-13 at 13 17 17](https://github.com/idan-gute/global-news-app/assets/104825246/e0837837-5e25-4771-8420-ff841fa4e066)